_projects/power_of_many.md

```diff
@@ -4,9 +4,21 @@ collection: projects
 permalink: /projects/power_of_many
 abstract: "Enabling ensemble programming for simulating and analyzing climatic models and seismic wave data."
 status: active
+logo: pom_logo.jpg
 ---
 
-This project brings together an interdisciplinary team of cyber-infrastructure and domain scientists to develop the [RADICAL Ensemble Toolkit (EnTK)](https://github.com/radical-cybertools/radical.entk). EnTK enhances the ability of scientists to effectively and efficiently study natural hazards and renewable energy. The use of EnTK on high performance computing resources supports novel “ensemble-based” methods at scales not currently possible.
-
-<a href="https://github.com/radical-collaboration/hpc-workflows"><i class="fa fa-github"></i>GitHub Repository</a><br>
-<a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1639694"><i class="fa fa-nsf"></i>NSF Award</a>
+<div class="project-description">
+	<figure>
+		<img src="../images/projects/pom.png" width="450">
```

Comment:
Here the user will not need to add the exact name. The page can pick it up from the header
```suggestion
		<img src="{{site.baseurl}}/images/projects/{{page.logo}}" width="250">
```